README.md

```diff
@@ -23,7 +23,8 @@ sonar-scala
 
 **SonarQube plugin for static code analysis of Scala projects.**
 
-Intended for [SonarQube 6.7+ LTS](https://www.sonarqube.org/downloads) and Scala 2.11/2.12.
+Intended for [SonarQube 6.7+ LTS](https://www.sonarqube.org/downloads) and Scala 2.11/2.12.  
+It does not support Windows.
```

Comment:
Uhm... maybe we should be a little bit more precise with this?

Something like:
> "Running analysis from a windows machine is not supported"

Also, I'm not sure if all the plugin fail, or only Scoverage?

Reply:
I updated this in 204ae17.

> Also, I'm not sure if all the plugin fail, or only Scoverage?

I'm not sure either - it's worth investigating this as a part of #65.